sdks/python/apache_beam/transforms/trigger.py

```diff
@@ -98,7 +94,11 @@ def with_prefix(self, prefix):
 
 
 class _CombiningValueStateTag(_StateTag):
-  """StateTag pointing to an element, accumulated with a combiner."""
+  """StateTag pointing to an element, accumulated with a combiner.
+
+  The given tag must be unique for this stage.  If CombineFn is None then
```

Comment:
And signing yourself up for a lifetime of `if combine_fn is not None` doesn't seem very pleasant...

Reply:
Removed that line.